reference/customers.sf.yml

```diff
@@ -57,7 +58,275 @@ paths:
                     - fieldId: '25'
                       fieldValue: Leave in backyard
         description: Data sent the the customer endpoint when creating a customer during checkout.
+  '/customers/{customerId}/metafields':
+    get:
+      summary: Get customer metafields.
+      tags:
+        - Customer Metafields
+      description: Get a customer's metafields.
+      operationId: getCustomersMetafields
+      parameters:
+        - $ref: '#/components/parameters/customerId'
+        - name: customerId
+          in: query
+          schema:
+            type: integer
+      requestBody:
+        required: true
+        content: 
+          application/json:
+            schema:
+             $ref: '#/components/schemas/metafield'
+      responses:
+        '200':
+          description: |
+            List of `Metafield` objects.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/responses/MetafieldResponse_Post'
+        '500':
+          description: Internal Server Error
+    post:
```

Comment:
we also have `put` and `delete` methods in batch endpoint, where list of customer metafields can be updated.
For batch update request body is an array with objects with following fields: 
- id (mandatory integer)
- namespace (optional String)
- key (optional String)
- value (optional String)
- permission_set (optional String)
- description (optional String)

Delete batch endpoints accepts array of integers that represent ids of metefields to delete.

Reply:
and example response for batch POST and PUT responses looks like this:
![image](https://github.com/bigcommerce/docs/assets/136105824/6245ac5d-6d50-4d00-afc2-229f6c0bdde7)

and for batch delete:
![image](https://github.com/bigcommerce/docs/assets/136105824/5607278f-e6ee-4367-b8f9-569490557a38)
(errors have the same structure as for POST and PUT.

Reply:
Sorry then, I wasn't aware of storefront customer REST so I assumed it is v3.

Reply:
but looks like you have described "Creates Customer metafields" which is batch create. I'm am a little confused now.